library/tasks/CreateCountMatrix.wdl

```diff
@@ -87,10 +87,11 @@ task CreateSparseCountMatrix {
   command {
     set -e
 
-    if [[ ${counting_mode} -eq "sc_rna" ]]
-    then
-      counting_mode_arg=" --sn-rna-seq-mode "
-    fi
+
```

Comment:
If this code is not needed, please remove

Reply:
This is a merge to the snrna-seq branch not the master. This code will likely be needed again when we get the whole pipeline working for snRNA-seq

Reply:
Sounds good